llama-index-integrations/vector_stores/llama-index-vector-store-firestore/llama_index/vector_store/firestore/base.py

```diff
@@ -0,0 +1,218 @@
+"""Firestore vector store index.
+
+An index that is built on top of an existing vector store.
+
+"""
+
+import logging
+from typing import Any, List, Optional, Union
+
+import more_itertools
+from google.cloud.firestore import Client
+from google.cloud.firestore_v1 import And, FieldFilter, Or
```

Comment:
I think we can import just from `google.cloud.firestore` 
```
from google.cloud.firestore import And, FieldFilter, Or
```

Do you get any error when trying that? (I don't have any problems importing like this, also would be worth checking the other imports)

Reply:
No issues doing that, thanks for the note.